CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 2.0.10
+
+* `Server` and related classes can now take an `onUnhandledError` callback to
```

Comment:
This is a new feature for clients to use. The version should be bumped to `2.1.0`

Reply:
Done